src/map/path.hpp

```diff
@@ -49,6 +49,10 @@ struct shootpath_data {
 #define check_distance_client_blxy(bl, x1, y1, distance) check_distance_client((bl)->x-(x1), (bl)->y-(y1), distance)
 #define check_distance_client_xy(x0, y0, x1, y1, distance) check_distance_client((x0)-(x1), (y0)-(y1), distance)
 
+#define distance_math_bl(bl1, bl2) distance_math((bl1)->x - (bl2)->x, (bl1)->y - (bl2)->y)
+#define distance_math_blxy(bl, x1, y1) distance_math((bl)->x-(x1), (bl)->y-(y1))
+#define distance_math_xy(x0, y0, x1, y1) distance_math((x0)-(x1), (y0)-(y1))
```

Comment:
`distance_math_blxy` and `distance_math_xy` are unused?

Reply:
Yes, I just wanted to make it consistent with the other macros, in case someone wants to use them later.